stream-compositions/api/service-api/payment-order-composition-api/pom.xml

```diff
@@ -0,0 +1,113 @@
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>service-api</artifactId>
+        <groupId>com.backbase.stream.compositions</groupId>
+        <version>3.1.0</version>
+    </parent>
+
+    <groupId>com.backbase.stream.compositions.api</groupId>
+    <artifactId>payment-order-composition-api</artifactId>
+
+    <name>Stream :: Compositions :: API :: Payment Order</name>
+
+    <properties>
+        <findMainClass.skip>true</findMainClass.skip>
+        <spring-boot.repackage.skip>true</spring-boot.repackage.skip>
+        <sonar.skip>true</sonar.skip>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.backbase</groupId>
+                <artifactId>backbase-bom</artifactId>
+                <version>${backbase-bom.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>assemble-api-zip</id>
+                        <goals>
+                            <goal>single</goal>
+                        </goals>
+                        <phase>package</phase>
+                        <configuration>
+                            <descriptors>
+                                <descriptor>assembly/api.xml</descriptor>
+                            </descriptors>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>unpack</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>com.backbase.dbs.paymentorder</groupId>
+                                    <artifactId>payment-order-service</artifactId>
+                                    <type>zip</type>
+                                    <classifier>api</classifier>
+                                    <outputDirectory>${project.build.directory}/yaml</outputDirectory>
+                                    <overWrite>true</overWrite>
+                                </artifactItem>
+                            </artifactItems>
```

Comment:
Do we need it ? Is it used in this module ?

Reply:
@piotr-backbase do you mean the payment-order-service dependency? We pull the service-api.yml from it and reference it. 

                    ```
<execution>
                        <id>boat-bundle</id>
                        <phase>generate-sources</phase>
                        <goals>
                            <goal>bundle</goal>
                        </goals>
                        <configuration>
                            <input>${project.basedir}/src/main/resources</input>
                            <output>${project.build.directory}</output>
                            <versionFileName>true</versionFileName>
                            <includes>*api*.yaml</includes>
                        </configuration>
                    </execution>
```

I'm going to keep that for now, unless you let me know otherwise.